FreeRTOS-Plus/Source/Application-Protocols/platform/freertos/mbedtls/mbedtls_freertos_port.c

```diff
@@ -0,0 +1,286 @@
+/*
```

Comment:
Were these platform files meant to be in this PR? Seems like an unrelated change and not part of the PR description

Reply:
I've updated the PR description